lib/mongo/error/notable.rb

```diff
@@ -47,7 +47,7 @@ def add_note(note)
 
       # @api public
       def message
-        super + notes_tail
```

Comment:
below, `to_s` is defined as:

```
def to_s
  super + notes_tail
end
```

`message` actually calls `to_s`, so this was causing `notes_tail` to be appended to the message twice.